docs/dev.md

```diff
@@ -64,6 +70,71 @@ dep ensure
 /go/bin/deepcopy-gen --output-base=/go/src --input-dirs=github.com/kubeflow/experimental-kvc/pkg/apis/cr/v1/...
 go build -gcflags "-N -l" github.com/kubeflow/experimental-kvc
 ```
+### Developing on a workstation
+This is the preferred method for developers who don't want to run `docker` locally and or don't mind setting up
+`Go` development environment on their workstation.
+
+Prior to using this method please ensure you have `go 1.9.2` or better development environment setup and running.
+
+For developing on `Linux` add the following lines to your `.bashrc`
+```bash
+export GOROOT="/usr/lib/go-1.9"
```

Comment:
Can we say, make sure your GOROOT, GOPATH and your PATH are set to the appropriate paths, for example on Ubuntu: xyz and on Mac: xyz?